crates/bevy_render/src/render_phase/draw_state.rs

```diff
@@ -95,6 +95,19 @@ impl DrawState {
     ) -> bool {
         self.index_buffer == Some((buffer, offset, index_format))
     }
+
+    /// Resets tracking state
+    pub fn reset_tracking(&mut self) {
```

Comment:
Couldn't this just do `self = Default::default()`

Reply:
It is important to keep vector lengths to device limits.

See `TrackedRenderPass::new`

```
 state: DrawState {
                bind_groups: vec![(None, Vec::new()); max_bind_groups],
                vertex_buffers: vec![None; max_vertex_buffers],
                ..default()
},
```